package-lock.json

```diff
@@ -1112,13 +1112,13 @@
     },
     "array-slice": {
       "version": "0.2.3",
-      "resolved": "https://registry.npmjs.org/array-slice/-/array-slice-0.2.3.tgz",
+      "resolved": "https://bams-aws.refinitiv.com/artifactory/api/npm/default.npm.global/array-slice/-/array-slice-0.2.3.tgz",
       "integrity": "sha1-3Tz7gO15c6dRF82sabC5nshhhvU=",
       "dev": true
     },
     "array-union": {
       "version": "1.0.2",
-      "resolved": "https://registry.npmjs.org/array-union/-/array-union-1.0.2.tgz",
+      "resolved": "https://bams-aws.refinitiv.com/artifactory/api/npm/default.npm.global/array-union/-/array-union-1.0.2.tgz",
```

Comment:
Could you please check the source of packages ? 
I think should be registry.npmjs.org